app/src/main/res/layout/item_campaign_large.xml

```diff
@@ -1,67 +1,111 @@
+
+            <ImageView
+                android:id="@+id/image"
+                android:layout_width="match_parent"
+                android:layout_height="@dimen/campaign_height_expanded"
+                android:scaleType="centerCrop"
+                android:src="@drawable/placeholder"/>
+
+            <View
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:background="@drawable/gradient"/>
+        </FrameLayout>
 
         <org.dosomething.letsdothis.ui.views.typeface.CustomTextView
             android:id="@+id/title"
-            tools:text="Title"
-            app:typeface="brandon_bold"
-            android:textAllCaps="true"
-            android:textSize="@dimen/text_48"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
             android:layout_above="@+id/call_to_action"
-            android:textColor="@android:color/white"
             android:layout_centerInParent="true"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"/>
+            android:textAllCaps="true"
+            android:textColor="@android:color/white"
+            android:textSize="@dimen/text_48"
+            app:typeface="brandon_bold"
+            tools:text="Title"/>
 
         <org.dosomething.letsdothis.ui.views.typeface.CustomTextView
-            tools:text="Call to action"
-            android:textSize="@dimen/text_33"
-            android:textColor="@android:color/white"
-            android:layout_centerInParent="true"
-            android:layout_alignParentBottom="true"
             android:id="@+id/call_to_action"
             android:layout_width="wrap_content"
-            android:layout_height="wrap_content"/>
+            android:layout_height="wrap_content"
+            android:layout_alignParentBottom="true"
+            android:layout_centerInParent="true"
+            android:textColor="@android:color/white"
+            android:textSize="@dimen/text_33"
+            tools:text="Call to action"/>
 
     </RelativeLayout>
 
     <org.dosomething.letsdothis.ui.views.typeface.CustomTextView
-        android:layout_below="@+id/image_and_title"
         android:id="@+id/problemFact"
-        tools:text="Problem fact"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"/>
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/image_and_title"
+        android:layout_margin="@dimen/padding_small"
+        tools:text="Problem fact"/>
 
     <!--FIXME we need some lightning here-->
-    <org.dosomething.letsdothis.ui.views.typeface.CustomButton
-        android:layout_below="@+id/problemFact"
-        android:id="@+id/campaign_details"
-        app:typeface="brandon_bold"
-        android:textAllCaps="true"
-        android:text="@string/stop_being_bored"
+
+
+    <RelativeLayout
```

Comment:
FIXME: The click listener should be on this view, not just the "Stop Being Bored" text.